input/fsh/BFDR/bundlescompositions/BundleDocumentBFDRNew.fsh

```diff
@@ -0,0 +1,15 @@
+Profile: BundleDocumentBFDRNew
+Parent: Bundle
+Title: "Bundle - Document Birth and Fetal Death"
+Description: "This Bundle profile represents a Birth and Fetal Death Document Bundle. It contains any one of the Birth and Fetal Death Compositions."
+* identifier 1.. 
+  * ^short = "BFDR Document Bundle identifier"
+* type = #document (exactly)
+* type 
+* entry 
+  * ^slicing.discriminator.type = #profile
+  * ^slicing.discriminator.path = "$this.resource"
+  * ^slicing.rules = #open
+* entry contains BFDRComposition 1..1 
+* entry[BFDRComposition].resource 1.. 
+* entry[BFDRComposition].resource only Composition
```

Comment:
Not sure if generalization is intentional here, but the original bundle references the 4 specific compositions: 

* resource only CompositionProviderLiveBirthReport or CompositionProviderFetalDeathReport or CompositionJurisdictionLiveBirthReport or CompositionJurisdictionFetalDeathReport